resources/views/components/event-listing.blade.php

```diff
@@ -39,9 +36,7 @@ class="relative inline-flex items-center px-4 py-2 border border-transparent tex
         <ul>
             @if ($event->hasSpeaker(1))
                 <x-event-list-speaker :event="$event" number="1"/>
-
             @endif
-
```

Comment:
Think it's best to revert much of the white lines you've removed in this PR. They're there to give the code some breathing room.